apps/livreGUI/CMakeLists.txt

```diff
@@ -16,7 +16,12 @@ if(NOT QT5WIDGETS_FOUND)
   return()
 endif()
 
+include(zerobufGenerateCxx)
+zerobuf_generate_qobject(ZEROBUF_GENERATED ${CMAKE_CURRENT_BINARY_DIR}
+  ${PROJECT_SOURCE_DIR}/livre/lib/zeroeq/volumeRendererParameters.fbs)
```

Comment:
There was already a zerobuf::VolumeRendererParameters, so shouldn't the generation be removed somewhere else?